frontend/locales/en.js

```diff
@@ -28,7 +28,7 @@ export default {
     documentation: 'Documentation',
     documentationText: 'Check out our docs',
     contribute: 'Contribute',
-    contributeText: 'Help us improve with issueing bugs and feature requests',
+    contributeText: 'Help us improve with issuing bugs and feature requests',
```

Comment:
Maybe should even be "submitting bug reports and feature requests". Thoguhts?

Reply:
I will try my best. I have sent Tim a mail about creating a forum/channel for beginners like me. I was trying to run the project but haven't been able to.
Everything was working fine but the room(whiteboard) is not being created.
Because of that couldn't run and couldn't find any bugs or feature request as of now.

Reply:
I meant, maybe the wording should be "submitting bug reports and feature requests".

Reply:
Sorry, I thought of mentioning that but couldn't understand what type of PR it would be as neither it was a bug nor a feature.
And I went through your website. lol.😂

Reply:
Hello I contacted Tim and he will reply soon @tripathi-abhishek

Reply:
thanks for your help guys!

Reply:
@tripathi-abhishek we created a discussions tab where you can explain your problems with the configuration of whitebird

Reply:
Thank you!